config/uberon.yml

```diff
@@ -21,18 +21,24 @@ entries:
 - exact: /phenoscape-ext.owl
   replacement: http://phenoscape.svn.sourceforge.net/svnroot/phenoscape/trunk/vocab/edit/phenoscape-ext.owl
 
-- exact: /composite-vertebrate.owl
+- exact: /releases/2021-10-01/composite-vertebrate.owl
```

Comment:
Why is this necessary? If you want `/composite-vertebrate.owl` to be dealt with by the fall-thru, what is the point of hard coding a standard redirect here? Why not just delete the whole goal?

Reply:
To keep this release version, no?

Reply:
- exact: /releases/2021-10-01/composite-vertebrate.owl
   replacement: https://github.com/obophenotype/uberon/releases/download/v2021-10-01/composite-vertebrate.owl

Redirects the release URL ending with `/releases/2021-10-01/composite-vertebrate.owl` to the release download page of the same version.  The following piece (later on in the makefile, will do the exact same thing, unless I am completely besides me..

```
- prefix: /releases/
  replacement: https://github.com/obophenotype/uberon/releases/download/v
```